### ISSUE TITLE
Shelter vs. Lineage OS 

### ISSUE THREAD

**Issue by Moostek** [PeterCxy/Shelter#100]
Hello there, i do not report here any bug, but i would like to know if there is anybody who might confirm me that the Shelter app is compatible with the Lineage OS (https://lineageos.org/) and not causing any troubles there? Because the authors of Shelter stated that _"the app depends on your Android system’s implementation of Work Profile"._ So i would like to know more about this in regards with Lineage OS. 

Thanks for your notes.

**Comment by jmichael2497** [comment]
please do not abuse the "Issue (for bug reporting)" system as this discourages the developers from regularly reviewing and responding to actual bug report issues.

to ask about compatibility with a particular OS, try asking in forums for that particular OS, or general user chat sites instead.

but also don't forget to spend at least 5 minutes trying a few general web search engines with different keywords, because the chances are pretty good your questions have already been asked and answered somewhere.

hope that helps, cheers.

**Comment by Moostek** [comment]
Yeah, i understand. But to gain some informations about some relation between OS and app is sometimes difficult. Both sides telling me 'try it on the other side'... I am sure you understand what i am trying to describe here. Of course i tried to search on internet (i found one thread, but not with the exact content about what i am looking for). I tried to ask on LOS forum about the same thing, but they have pretty strict rules in discussions (asking about 3rd party products). 

I wish that free software world communicates better and not the way that the only suggestion which left is "find it yourself somewhere on internet". I am an end user and i want/need a help. And producer's websites/forums they are many times the only chance and source where to get right informations. 

Maybe it will be good - and it makes sence to me - to establish here a thread called "Questions and functionality". This sounds as a solution, because such thread will give a place for questioning about the app in relation with anything, and the panel "Issues" will not be "misused and undermined".

**Comment by nr458h** [comment]
To give a helping hand. I have shelter on two different LineageOS devices and didn't run into problems till now.
So I think you can try and then report and close the issue.

**Comment by Moostek** [comment]
Hello, thanks for your comments and tips for LOS. So i will try this. 
And is there some manual or FAQ how to use Shelter? 
How does it work i.e. is it possible to install the app right inside the Shelter or must it be installed on OS first? 

I really think that it will be fine if there will be some thread for questions about the app.

**Comment by nr458h** [comment]
@Moostek 
Maybe you can post your experiences with shelter and or close this issue. Thanks for keeping the issue list clean.@Moostek
